server/neptune/workflows/internal/deploy/revision/queue/worker.go

```diff
@@ -9,12 +9,19 @@ import (
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/activities"
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/config/logger"
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/terraform"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/github"
 	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/root"
 	"go.temporal.io/sdk/temporal"
 	"go.temporal.io/sdk/workflow"
 )
 
 const DeploymentInfoVersion = "1.0.0"
+const UpdateCheckrunRetryCount = 5
+
+const (
+	IdenticalRevisonSummary = "This revision is identical to the current revision and will not be deployed"
```

Comment:
Why are we preventing this? That means we'd be preventing redeploys which we'd want probably want to support without asking people to merge a noop commit

Reply:
Ahh true, didn't think of this use case! I have updated it now!